RollingPaper/RollingPaper/ViewControllers/Core/PaperTemplate/PaperSettingViewController.swift

```diff
@@ -160,6 +165,13 @@ final class PaperSettingViewController: UIViewController {
                 switch event {
                 case .timePickerChange(let time):
                     self.timePickerButton.setTitle(time, for: .normal)
+                    if time == self.noTimeText {
```

Comment:
해당 부분도 위에 처럼 case로 만든 뒤에 wherer을 통해서 조건을 거는건 어떨까요?
```suggestion
                    case time where case == noTimetext
```
참고자료 -> https://seungchan.tistory.com/83